test/BDNPerfTests/ConfigFiles/CI_CONFIG_BDN_Benchmark_Operations.BasicOperations.json

```diff
@@ -0,0 +1,17 @@
+{
+    "configuration": "Release",
```

Comment:
Merge everything into a single JSON file called bdn-allocated-targets.json with one nested section for each filter.

Reply:
Not sure I understand why to do this?  I assume this is just an extra task near the end of the YML to do so we have it in the artifacts or are you talking about doing it instead of having separate json files?  I am not sure how the merge works but can you just continually merge them together as more tests finish? For example ... you concurrently start 10 BDN runs and #3 finishes, then #4 is finishing so at the end of that it merges, 3 and 4 together, the another test (7) finishes and so that merges 3, 4, 7 together etc until all 10 are done?  I hope it doesn't just add one and it is an actual merge. I can look into it.

Reply:
i was referring to the INPUT config file, not the output files. Output files have to definitely be different, not possible to merge correctly in general, so let us not even try that.

Reply:
ok - can resolve this as the Input config is already being discussed